src/MetaModels/DcGeneral/Events/Table/Attribute/Subscriber.php

```diff
@@ -519,24 +515,4 @@ public function handleUpdateAttribute(PostPersistModelEvent $event)
             }
         }
     }
-
-    /**
-     * Handle the deletion of an attribute and all attached data.
-     *
-     * @param PreDeleteModelEvent $event The event.
-     *
-     * @return void
-     */
-    public function handleDeleteAttribute(PreDeleteModelEvent $event)
-    {
-        if (($event->getEnvironment()->getDataDefinition()->getName() !== 'tl_metamodel_attribute')) {
-            return;
-        }
-
-        $attribute = $this->createAttributeInstance($event->getModel()->getPropertiesAsArray());
-
-        if ($attribute) {
-            $attribute->destroyAUX();
```

Comment:
o.k. you missed https://github.com/MetaModels/core/blob/master/src/MetaModels/Attribute/BaseSimple.php#L194